app/styles/individu-form.scss

```diff
@@ -0,0 +1,14 @@
+@import 'commons';
+
+#warning-less-18 {
+    margin-left:18%;
+    margin-right:18%;
+    text-align: justify;
+}
+
+@media (max-width: $screen-sm-min) {
+    #warning-less-18 {
+        margin-left:0;
+        margin-right:0;
+    }
+}
```

Comment:
Sorry for introducing `scss`, but I don't know how to do it in pure `css`, and I don't really have time to dig more. 
Feel free to convert it if you like.